src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+let total = 0;
+
+for (const countriePopulation of population) {
```

Comment:
Typo: change `countriePopulation` to `countryPopulation` for better readability.

Reply:
Consider using functional programming methods like reduce instead of a traditional loop.